### PR TITLE
Upd pymod

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b upd-pymod
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/models/rnafusion/rnafusion.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class RnafusionParameters(WorkflowParameters):
     fusioncatcher: bool = True
     starfusion: bool = True
     trim_tail: int = 50
-    clusterOptions: str = Field(..., alias="cluster_options")
+    clusterOptions: str = Field(None, alias="cluster_options")
```
Comment (diitaz93) [inline cg/models/rnafusion/rnafusion.py:39]:
```suggestion
    clusterOptions: str | None = Field(None, alias="cluster_options")
```

**Review by diitaz93 — APPROVED** [review]
👍

**Comment by henrikstranneheim** [comment]
### Start rnafusion, 👍 
```
home/proj/stage/bin/miniconda3/envs/S_cg/lib/python3.11/site-packages/pydantic/_internal/_config.py:334: UserWarning: Valid config keys have changed in V2:
* 'fields' has been removed
  warnings.warn(message, UserWarning)
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Called undefined __pydantic_serializer__ on HousekeeperAPI, please wrap
Called undefined __dataclass_fields__ on HousekeeperAPI, please wrap
Starting analysis for cuddlyhen
Case cuddlyhen exists in Status DB
Illumina run check is not applicable - the case is either down sampled or external.
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Check if FASTQ pair exists
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Both /home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001_R1_001.fastq.gz and /home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001_R2_001.fastq.gz from sample surelyrightserval are already in Housekeeper
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Check if FASTQ pair exists
Check if pending compression file exists
/home/proj/stage/housekeeper-bundles/surelyrightserval/2023-02-06/HCKLTDSX2_ACC8869A1_S6_L001.crunchy.pending.txt does not exist
Compression/decompression is not running
Dry run: Config files will not be written
Case cuddlyhen exists in Status DB
Samples linked to case cuddlyhen: 1
Case cuddlyhen exists in Status DB
Running analysis for cuddlyhen
Workflow will be executed using Tower
Running command /home/proj/stage/bin/tw launch --work-dir /home/proj/stage/rnafusion/cases/cuddlyhen/work --profile singularity,hasta --params-file /home/proj/stage/rnafusion/cases/cuddlyhen/cuddlyhen_params_file.yaml --config /home/proj/stage/rnafusion/cases/cuddlyhen/cuddlyhen_nextflow_config.json --name cuddlyhen --revision 3.0.1 --compute-env hasta-normal nfcore_rnafusion
Dry run: process call will not be executed!!

Dry-run: Action running would be set for case cuddlyhen

File: cg/models/rnafusion/rnafusion.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg/models/rnafusion/rnafusion.py]: Switched to just str

**Comment by henrikstranneheim** [comment]
Deployed to prod
```
Log deploy... done.
cg, version 63.5.2

**Pull request merged**